src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
```

Comment:
You should use variables in places like this. :)
2nd point in [checklist.](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)

Something like that
```
const LONG_TERM = 7;
const LONG_TERM_DISCOUNT = 50;

if (numberOfDays >= LONG_TERM) {
  return basePrice - LONG_TERM_DISCOUNT;
}
```

Not : 
```
if (numberOfDays >= 7) {
  return basePrice - 50;
}
```